src/uxbox/frontend/routes.clj

```diff
@@ -75,8 +75,8 @@
        ;; Images
        [:delete "library/images/:id" #'images/delete-image]
        [:put "library/images/:id" #'images/update-image]
-       [:get "library/images" #'images/list-images]
-       [:post "library/images" #'images/create-image]
+       [:get "library/images/:collid" #'images/list-images]
```

Comment:
Seems like you are using a wrong enpoint for list images by collection. I think you should use the `library/image-collections/:id/images` endpoint for that.